plugins/obs-ffmpeg/obs-ffmpeg-source.c

```diff
@@ -491,17 +514,22 @@ static void ffmpeg_source_update(void *data, obs_data_t *settings)
 	if (s->speed_percent < 1 || s->speed_percent > 200)
 		s->speed_percent = 100;
 
-	if (s->media) {
+	if (s->media && should_restart_media) {
 		media_playback_destroy(s->media);
 		s->media = NULL;
 	}
 
 	bool active = obs_source_active(s->source);
-	if (!s->close_when_inactive || active)
+
+	if (s->media) { // directly set options if media is playing
```

Comment:
I'd prefer the comment on the line above, and I'd prefer `/* */` style comments for C code. I know we've been inconsistent on this in the past though. I've just kind of starting thinking about this just recently after seeing it and I'd really like to have consistency with kernel normal form styling.